assets/src/components/amp-setting-toggle/index.js

```diff
@@ -49,5 +56,8 @@ AMPSettingToggle.propTypes = {
 	disabled: PropTypes.bool,
 	onChange: PropTypes.func.isRequired,
 	text: PropTypes.string,
-	title: PropTypes.string.isRequired,
+	title: PropTypes.oneOfType( [
+		PropTypes.string,
+		PropTypes.elementType,
+	] ).isRequired,
```

Comment:
@pierlon Actually, it seems this doesn't work:

![image](https://user-images.githubusercontent.com/134745/87737141-0963af00-c78f-11ea-943f-66b57d5aa3fb.png)

Reply:
`PropTypes.node` will work.

Reply:
Instead of the `oneOfType`

Reply:
Or as one of the `oneOfType`s  😉

Reply:
Done in ~13e29ff~ 17daeec 😊